game/gui.rpy

```diff
@@ -193,7 +193,7 @@ define -2 gui.confirm_button_text_xalign = 0.5
 define -2 gui.page_button_borders = Borders(10, 4, 10, 4)
 
 
-define -2 gui.quick_button_text_size = 16
+define -2 gui.quick_button_text_size = 19
```

Comment:
I'm not sure that the text of the Russian translation will fit in now...

Reply:
Ohhh, I didn't think about translations, but it might not be a big deal.